### DISCUSSION TITLE
"Metadata" / Properties - Details doesn't transfer and gets locked on original

### DISCUSSION THREAD

**Discussion by rodvinsky** [mifi/lossless-cut#1133] [Support]
I was hoping that the "Preserve all MP4/MOV metadata?" meant that the information in the Properties panel under the Details tab would be copied over, but I guess that's a different kind of metadata. 

I also noticed that these details in the original are no longer editable after exporting. Does anyone know if this can be fixed?

And does anyone know a more efficient way of marking the exported videos with titles, comments, tags etc. than having to do it over again?

**Comment by mifi** [comment]
Issues around metadata have been discussed extensively. Unfortunately it's not so easy because ffmpeg has a tendency to handle this inconsistently. I collected a summary of information in this issue: #1027

**Comment by reefhound** [comment]
Any update on when exiftool will be integrated into the app? I continue to have problems with metadata not being copied into the clips either completely or correctly. I've been using exiftool in a batch file to fix things but this is an annoying additional step.

exiftool -tagsFromFile original.mp4 -all:all -overwrite_original with-exiftool.mp4

More specifically, after creating a clip much of the metadata is copies but cannot be seen in File Explorer because the majorbrand and minorversion have been altered. In the exiftool output below, you can see the clip was generated with Major Brand as "mp41". Using exiftool as above, resets it to the original and it displays the metadata. 

I can also use ffmpeg to copy and it won't use the original values of major/minor but will set Major to "MP4 Base Media v1 [IS0 14496-12:2003]" and Minor to "0.2.0". That also displays fine.

F:\Videos>exiftool -title -mimetype -compatiblebrands -majorbrand -minorversion *.mp4
======== GH011086-00.00.01.001-00.00.20.020.MP4
Title                           : Gray angel exploring the bottom
MIME Type                       : video/mp4
Compatible Brands               : mp41
Major Brand                     : mp41
Minor Version                   : 538120216
======== GH011086.MP4
Title                           : Gray angel exploring the bottom
MIME Type                       : video/mp4
Compatible Brands               : mp41
Major Brand                     : MP4 v1 [ISO 14496-1:ch13]
Minor Version                   : 2013.10.18
